modal-go/queue.go

```diff
@@ -145,11 +145,14 @@ func QueueLookup(ctx context.Context, name string, options *LookupOptions) (*Que
 
 // QueueDelete removes a queue by name.
 func QueueDelete(ctx context.Context, name string, options *DeleteOptions) error {
+	if options == nil {
+		options = &DeleteOptions{}
+	}
 	q, err := QueueLookup(ctx, name, &LookupOptions{Environment: options.Environment})
 	if err != nil {
 		return err
 	}
-	_, err = client.QueueDelete(ctx, pb.QueueDeleteRequest_builder{QueueId: q.QueueId}.Build())
+	_, err = client.QueueDelete(q.ctx, pb.QueueDeleteRequest_builder{QueueId: q.QueueId}.Build())
```

Comment:
Would love your eyes on this @ekzhang , feels a little hairy (see the commit 7e939204e5eec9b6b2d80a92d1e4c2af6e881c30).